attestation-service/verifier/src/se/mod.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) Copyright IBM Corp. 2024
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::*;
+use crate::se::ibmse::SeAttestationClaims;
+use crate::{InitDataHash, ReportData};
+use async_trait::async_trait;
+use std::fs;
+
+pub mod ibmse;
+
+const SE_HOST_KEY_DOCUMENTS_ROOT: &str = "/run/confidential-containers/ibmse/hkds";
+const SE_CERTIFICATES_ROOT: &str = "/run/confidential-containers/ibmse/certs";
+const SE_CERTIFICATE_ROOT_CA: &str = "/run/confidential-containers/ibmse/certs/ca";
+const SE_CERTIFICATE_REVOCATION_LISTS_ROOT: &str = "/run/confidential-containers/ibmse/crls";
+const SE_IMAGE_HEADER_FILE: &str = "/run/confidential-containers/ibmse/hdr/hdr.bin";
+const SE_MEASUREMENT_ENCR_KEY_PRIVATE: &str =
+    "/run/confidential-containers/ibmse/rsa/encrypt_key.pem";
+const SE_MEASUREMENT_ENCR_KEY_PUBLIC: &str =
```

Comment:
Keep in mind that these files are set-up on the trustee side -- Trustee runs on a trusted place and does not need to be in TEE. So we'd better make the files configurable. `/run` might not be a good place.

Reply:
Fixed, thanks!